build/dockerfiles/Dockerfile

```diff
@@ -76,30 +98,43 @@ RUN chmod g+rwx -R /mnt/rootfs/var/www/html/v3
 FROM scratch
 COPY --from=ubi-builder /mnt/rootfs/ /
 # Add UTF-8 for the database
-RUN localedef -f UTF-8 -i en_US en_US.UTF-8 && \
-    usermod -a -G apache,root,postgres postgres
+RUN ARCH=$(uname -m) && \
+    localedef -f UTF-8 -i en_US en_US.UTF-8 && \
+    if grep -q "^postgres:" /etc/group; then \
+        usermod -a -G apache,root,postgres postgres; \
+    else \
+        groupadd -r postgres && \
+        useradd -r -g postgres postgres && \
+        usermod -a -G apache,root postgres; \
+    fi
+
+RUN chmod g+w /entrypoint.sh
 USER postgres
 ENV LC_ALL=en_US.UTF-8 \
     LANG=en_US.UTF-8 \
     LANGUAGE=en_US.UTF-8 \
-    PGDATA=/var/lib/pgsql/15/data/database \
```

Comment:
We need this ENV, without it entrypoint.sh failed with `cp: cannot stat '/var/lib/pgsql/15/data/old': No such file or directory`